From 83a6f7e0422903535ba98c136351c1b34bc5c1b9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9o=20Cordeiro?=
 <75858274+CleoCordeiro@users.noreply.github.com>
Date: Tue, 2 Aug 2022 21:17:49 -0300
Subject: [PATCH] Update alignment.py

fixed imports of collections for different versions of python
---
 sepp/alignment.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/sepp/alignment.py b/sepp/alignment.py
index ada5de8..c847d95 100644
--- a/sepp/alignment.py
+++ b/sepp/alignment.py
@@ -26,7 +26,11 @@
 
 from sepp.filemgr import open_with_intermediates
 
-from collections import Mapping
+try:
+    from collections.abc import Mapping  # noqa
+except ImportError:
+    from collections import Mapping 
+    
 import copy
 from sepp import get_logger
 import io
